.tekton/pipeline-service-upgrade-test.yaml

```diff
@@ -10,20 +10,15 @@ metadata:
     pipelinesascode.tekton.dev/task: "[.tekton/tasks/setup-ci-runner-container.yaml]"
     pipelinesascode.tekton.dev/task-1: "[.tekton/tasks/deploy-cluster.yaml]"
     pipelinesascode.tekton.dev/task-2: "[.tekton/tasks/destroy-cluster.yaml]"
-    pipelinesascode.tekton.dev/task-3: "[.tekton/tasks/destroy-existing-cluster.yaml]"
-    pipelinesascode.tekton.dev/task-4: "[.tekton/tasks/generate-cluster-name.yaml]"
-    pipelinesascode.tekton.dev/task-5: "[.tekton/tasks/deploy-pipeline-service.yaml]"
-    pipelinesascode.tekton.dev/task-6: "[.tekton/tasks/test-pipeline-service.yaml]"
-    pipelinesascode.tekton.dev/task-7: "[git-clone]"
+    pipelinesascode.tekton.dev/task-3: "[.tekton/tasks/generate-cluster-name.yaml]"
+    pipelinesascode.tekton.dev/task-4: "[.tekton/tasks/deploy-pipeline-service.yaml]"
+    pipelinesascode.tekton.dev/task-5: "[.tekton/tasks/test-pipeline-service.yaml]"
+    pipelinesascode.tekton.dev/task-6: "[git-clone]"
     pipelinesascode.tekton.dev/max-keep-runs: "5"
 spec:
   pipelineRef:
     name: test-pipeline-service-upgrade
   params:
-    - name: aws_region
-      value: "us-east-1"
-    - name: ocp_version
```

Comment:
Why don't we keep the version as an explicit param?

Reply:
I would like to define them in one place, such as task [`deploy-cluster`](https://github.com/openshift-pipelines/pipeline-service/blob/2f35a207fed5a79a1116c35390eb7b83951bc0b5/.tekton/tasks/deploy-cluster.yaml#L8-L13), other pipelines can reference this task and keep the same version and AWS region